src/components/Nav/Nav.js

```diff
@@ -1,8 +1,44 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './Nav.scss';
 
-const Nav = () => {
-  return <h1>Nav</h1>;
-};
-
-export default Nav;
+export default function Nav() {
+  return (
+    <header className="nav">
+      <nav className="nav__bar">
+        <div className="nav__bar__left">
+          <Link to="/store" className="nav__bar__left__store">
+            <p>스토어</p>
+          </Link>
+        </div>
+        <div className="nav__bar__logo">
+          <Link to="/main" className="nav__bar__left__store">
+            <p>Health Eat</p>
+          </Link>
```

Comment:
```suggestion
          <Link to="/main" className="nav__bar__left__store">
            Health Eat
          </Link>
```
이렇게 수정하시고 스타일도 여기에 맞게 수정해주세요!

Reply:
수정 완료!